app/controllers/ems_common.rb

```diff
@@ -809,6 +826,9 @@ def set_record_vars(ems, mode = nil)
     if ems.supports_authentication?(:ssh_keypair) && !@edit[:new][:ssh_keypair_userid].blank?
       creds[:ssh_keypair] = {:userid => @edit[:new][:ssh_keypair_userid], :auth_key => @edit[:new][:ssh_keypair_password]}
     end
+    if ems.supports_authentication?(:bearer) && !@edit[:new][:bearer_token].blank?
+      creds[:bearer] = {:auth_key => @edit[:new][:bearer_token], :userid => "_"} # Must have userid
```

Comment:
Minor: since we have a specific type of auth I think we can try to have a more meaningful `:userid` here, e.g. "bearer". @Fryguy thoughts?  (@chessbyte let's not block the merge on this comment though)